### DISCUSSION TITLE
[DONE] Enable Facebook login handler to be called from outside Meteor

### DISCUSSION THREAD

**Discussion by diavrank** [meteor/meteor#11599] [Feature Requests]
Hello, I am using Meteor as a back-end for a **React Native App**. I have configured **OAuth** with **Facebook** and **Google** successfully (based on these articles for [Facebook](https://medium.com/differential/react-native-meteor-oauth-with-facebook-3d1346d7cdb7) and [Google](https://spencer-carli.medium.com/meteor-google-oauth-from-react-native-e30d146dfad6)).

Currently, the package of **accounts-google** already includes a **login handler** for Google, whereas **accounts-facebook** doesn't include it. So, I would like to make a PR to add this login handler for the Facebook package to make **easier** the process about **Login with Facebook** using Meteor and React Native. Actually, this is part of a [feature](https://github.com/TheRealNate/meteor-react-native/issues/36) from **react-native-meteor** package.

I read [here](https://github.com/meteor/meteor/pull/9150#issuecomment-334170848) that I have to post the idea first and then receive feedback about this. So, I will be waiting for the approval to create the PR.

**Comment by StorytellerCZ** [comment]
I would say go ahead. If it is missing, then adding a feature parity is fine.

**Reply by diavrank** [reply]
ok, I'll proceed with that.

**Comment by StorytellerCZ** [comment]
Will be released in Meteor 2.4 under `facebook-oauth@1.10.0`
